_posts/2019/2019-01-13-use-multiple-migrations-when-adding-database-constraints.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Use Multiple Migrations When Adding Database Constraints
+description: You need to avoid a long table rebuild and a write-lock
+layout: article
+category: ruby
+image:
+  base: '2018/use-multiple-migrations-when-adding-database-constraints'
+  alt: 'Locks'
+  source: 'https://unsplash.com/photos/8_NI1WTqCGY'
+  credit: 'Marcos Mayer'
+---
+
+Adding constraints to your application at the database layer is a good idea as it provides an extra layer of quality control on top of the data powering your application.
+
+One way to do that is to add default values or constraints (like making sure a field can’t be blank), this is easy to do with Rails’ migrations. But beware...
```

Comment:
Full stop after the closing parentheses. Then: "This is easy to do with Rails' migrations..."